bundlewrap/items/kubernetes.py

```diff
@@ -242,34 +258,7 @@ def get_auto_deps(self, items):
 
     @property
     def KIND(self):
-        name = self.name.split("/", 2)[1]
-        if name.lower() in (
-            "clusterrole",
-            "clusterrolebinding",
-            "configmap",
-            "cronjob",
-            "customresourcedefinition",
-            "daemonset",
-            "deployment",
-            "ingress",
-            "namespace",
-            "persistentvolumeclaim",
-            "service",
-            "secret",
-            "statefulset",
-        ):
-            raise BundleError(_(
-                "Kind of {item_type}:{name} (bundle '{bundle}') "
-                "on {node} clashes with builtin k8s_* item"
-            ).format(
-                item_type=self.ITEM_TYPE_NAME,
-                name=self.name,
-                bundle=self.bundle.name,
-                node=self.bundle.node.name,
-                regex=self.NAME_REGEX,
-            ))
-        else:
-            return name
+        return self.name.split("/", 2)[1]
 
     @property
     def resource_name(self):
```

Comment:
This method should be removed, too. Otherwise, we don’t check the actual k8s object name but just it’s bw name.